docs/source/daml/reference/exceptions.rst

```diff
@@ -0,0 +1,89 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _exceptions:
+
+Exceptions
+##########
+
+Exceptions are a Daml feature which allows provides a way to handle
+certain errors that arise during interpretation instead of aborting
+the transaction and rollback changes that happened before the
```

Comment:
The "and rollback changes" phrase sounds like it is still part of the instead clause.
Not sure what the best way to disambiguate this is, but here is a suggestion:

```suggestion
the transaction, and to roll back the state changes that lead to the
```